homeworks/hw1/BWT/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace BWT;
+
+using System;
+
+class Programm
```

Comment:
```suggestion
class Program
```
В английском "программа" пишется с одной "м".